pcsx2/Counters.cpp

```diff
@@ -270,6 +270,30 @@ static const char* ReportVideoMode()
 	}
 }
 
+Fixed100 GetVerticalFrequency()
+{
+	switch (gsVideoMode)
+	{
+	case GS_VideoMode::Uninitialized: // SYSCALL instruction hasn't executed yet, give some temporary values.
+		return 60;
+	case GS_VideoMode::PAL:
+		return EmuConfig.GS.FrameratePAL;
+	case GS_VideoMode::NTSC:
+		return EmuConfig.GS.FramerateNTSC;
+	case GS_VideoMode::HDTV_480P:
+		return 59.94;
+	case GS_VideoMode::HDTV_1080P:
+	case GS_VideoMode::HDTV_1080I:
```

Comment:
> HDTV 480p -> Vertical frequency of 59.94Hz and a frame frequency of 29.97 fps
> Other progressive modes -> Vertical frequency of 60Hz and a frame frequency of 30 fps

@ssakash you've made mistake, in all **progressive** modes vertical frequency is equal to frame frequency.